wcc-contentful-app/db/migrate/20181029185749_create_wcc_contentful_app_contact_form_submissions.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateWCCContentfulAppContactFormSubmissions < ActiveRecord::Migration[5.2]
```

Comment:
How does this migration get loaded?  Does rails automatically pick it up from the gem, or does it get dropped in `db/migrate` by the generator?